client/nsxt_client.go

```diff
@@ -293,3 +293,46 @@ func (c *nsxtClient) GetLoadBalancerStatus(loadBalancerID string) (loadbalancer.
 	loadBalancerStatus, _, err := c.apiClient.ServicesApi.ReadLoadBalancerServiceStatus(c.apiClient.Context, loadBalancerID, nil)
 	return loadBalancerStatus, err
 }
+
+func (c *nsxtClient) ListAllSections() ([]manager.FirewallSection, error) {
+	var firewallSections []manager.FirewallSection
+	var cursor string
+	for {
+		localVarOptionals := make(map[string]interface{})
+		localVarOptionals["cursor"] = cursor
+		firewallSectionsResult, _, err := c.apiClient.ServicesApi.ListSections(c.apiClient.Context, localVarOptionals)
+		if err != nil {
+			return nil, err
+		}
+		firewallSections = append(firewallSections, firewallSectionsResult.Results...)
+		cursor = firewallSectionsResult.Cursor
+		if len(cursor) == 0 {
+			break
+		}
+	}
+	return firewallSections, nil
+}
+
+func (c *nsxtClient) GetAllRules(sectionID string) ([]manager.FirewallRule, error) {
```

Comment:
This method name to GetAllFirewallRules as we have multiple rules

Reply:
fixed in 8aef830